### PR TITLE
[new release] metrics, metrics-unix and metrics-lwt (0.1.0)

### PR DESCRIPTION
**Pull request by samoht**
Metrics infrastructure for OCaml

- Project page: <a href="https://github.com/mirage/metrics">https://github.com/mirage/metrics</a>
- Documentation: <a href="https://mirage.github.io/metrics/">https://mirage.github.io/metrics/</a>

##### CHANGES:

Initial version

### CODE REVIEW

File: packages/metrics-unix/metrics-unix.0.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name] {with-test}
+]
+
+depends: [
+  "dune" {build}
+  "uuidm"
+  "metrics"
+  "mtime"
```
Comment (samoht) [inline packages/metrics-unix/metrics-unix.0.1.0/opam]:
```suggestion
  "mtime"
  "metrics-lwt" {with-test}
```

**Comment by kit-ty-kate** [comment]
```
#=== ERROR while compiling metrics-unix.0.1.0 =================================#
# context              2.0.0 | linux/x86_64 | ocaml-base-compiler.4.07.0 | pinned(https://github.com/mirage/metrics/releases/download/0.1.0/metrics-0.1.0.tbz)
# path                 ~/.opam/4.07/.opam-switch/build/metrics-unix.0.1.0
# command              ~/.opam/4.07/bin/dune build -p metrics-unix -j 127
# exit-code            1
# env-file             ~/.opam/log/metrics-unix-19-d71bab.env
# output-file          ~/.opam/log/metrics-unix-19-d71bab.out
### output ###
# File "src/unix/dune", line 5, characters 15-23:
# 5 |   (libraries   lwt.unix metrics fmt uuidm unix mtime mtime.clock
#                    ^^^^^^^^
# Error: Library "lwt.unix" not found.
# Hint: try: dune external-lib-deps --missing -p metrics-unix @@default
```

File: packages/metrics-unix/metrics-unix.0.1.0/opam [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  "metrics-lwt" {with-test}
```
Comment (samoht) [inline packages/metrics-unix/metrics-unix.0.1.0/opam]:
```suggestion
  "lwt"
  "metrics-lwt" {with-test}
```

File: packages/metrics-unix/metrics-unix.0.1.0/opam [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name] {with-test}
+]
+
+depends: [
+  "dune" {build}
+  "uuidm"
+  "metrics"
+  "mtime"
+  "lwt"
```
Comment (samoht) [inline packages/metrics-unix/metrics-unix.0.1.0/opam]:
```suggestion
  "lwt"
  "conf-gnuplot"
```

**Comment by mseri** [comment]
Looks good now. OpenSUSE is failing due to lack of gnuplot, do you want to do something about it or you think it is good to merge?

**Comment by mseri** [comment]
All green now. Thanks

**Pull request merged**